ukat/data/fetch.py

```diff
@@ -1,178 +1,589 @@
+                             ['c76cdc72e32ad41cb5c469a9ada5cb83',
+                              'fb9943f4a905c28a098b15194ffe2e61',
+                              '3890e970e58825983acdbfd8f07fa55d',
+                              'a536341625a299743557e467772a3e46'],
+                             doc='Downloading GE DWI data')
+
+fetch_dwi_philips = _make_fetcher('fetch_dwi_philips',
+                                  pjoin(ukat_home, 'dwi_philips'),
+                                  'https://zenodo.org/record/4757139/files/',
+                                  ['03901__DWI_5slices.nii.gz',
+                                   '03901__DWI_5slices.json',
+                                   '03901__DWI_5slices.bval',
+                                   '03901__DWI_5slices.bvec'],
+                                  ['03901__DWI_5slices.nii.gz',
+                                   '03901__DWI_5slices.json',
+                                   '03901__DWI_5slices.bval',
+                                   '03901__DWI_5slices.bvec'],
+                                  ['da96320b38c6b201cb858057b4b534b3',
+                                   '229e04a00fb4336a47af603ff565dea9',
+                                   '6816ed33bd087ef465bfde5a75c0c11b',
+                                   'c70d2a49c003dd53d63b9d4cb9388cdb'],
+                                  doc='Downloading Philips DWI data')
+
+fetch_dwi_siemens = _make_fetcher('fetch_dwi_siemens',
+                                  pjoin(ukat_home, 'dwi_siemens'),
+                                  'https://zenodo.org/record/4757887/files/',
+                                  ['00042__trig_dwi_13b_06dir.nii.gz',
+                                   '00042__trig_dwi_13b_06dir.json',
+                                   '00042__trig_dwi_13b_06dir.bval',
+                                   '00042__trig_dwi_13b_06dir.bvec'],
+                                  ['00042__trig_dwi_13b_06dir.nii.gz',
+                                   '00042__trig_dwi_13b_06dir.json',
+                                   '00042__trig_dwi_13b_06dir.bval',
+                                   '00042__trig_dwi_13b_06dir.bvec'],
+                                  ['1836b56ba028b5d5d41ae5f35313889a',
+                                   'a043b4fb0721d3c38db462d433975d31',
+                                   'a57ce54e88c154d06a34722eaabb60fb',
+                                   '32d551e73ab6481972a6c8eab44f556d'],
+                                  doc='Downloading Siemens DWI data')
+
+fetch_t1_philips_1 = _make_fetcher('fetch_t1_philips_1',
+                                   pjoin(ukat_home, 't1_philips_1'),
+                                   'https://zenodo.org/record/4762285/files/',
+                                   ['philips_1.zip'],
+                                   ['philips_1.zip'],
+                                   ['3928d498b30d811910856e82ac4d406f'],
+                                   unzip=True,
```

Comment:
So this unzip flag allows that we can upload compressed and uncompressed files without worrying?